lib/visitor-keys.js

```diff
@@ -1,289 +1,300 @@
-    "JSXSpreadAttribute": [
+    JSXSpreadAttribute: [
         "argument"
     ],
-    "JSXText": [],
-    "JSXFragment": [
+    JSXText: [],
+    JSXFragment: [
         "openingFragment",
         "children",
         "closingFragment"
     ],
-    "Literal": [],
-    "LabeledStatement": [
+    Literal: [],
+    LabeledStatement: [
         "label",
         "body"
     ],
-    "LogicalExpression": [
+    LogicalExpression: [
         "left",
         "right"
     ],
-    "MemberExpression": [
+    MemberExpression: [
         "object",
         "property"
     ],
-    "MetaProperty": [
+    MetaProperty: [
         "meta",
         "property"
     ],
-    "MethodDefinition": [
+    MethodDefinition: [
         "key",
         "value"
     ],
-    "NewExpression": [
+    NewExpression: [
         "callee",
         "arguments"
     ],
-    "ObjectExpression": [
+    ObjectExpression: [
         "properties"
     ],
-    "ObjectPattern": [
+    ObjectPattern: [
         "properties"
     ],
-    "PrivateIdentifier": [],
-    "Program": [
+    PrivateIdentifier: [],
+    Program: [
         "body"
     ],
-    "Property": [
+    Property: [
         "key",
         "value"
     ],
-    "PropertyDefinition": [
+    PropertyDefinition: [
         "key",
         "value"
     ],
-    "RestElement": [
+    RestElement: [
         "argument"
     ],
-    "ReturnStatement": [
+    ReturnStatement: [
         "argument"
     ],
-    "SequenceExpression": [
+    SequenceExpression: [
         "expressions"
     ],
-    "SpreadElement": [
+    SpreadElement: [
         "argument"
     ],
-    "Super": [],
-    "SwitchStatement": [
+    Super: [],
+    SwitchStatement: [
         "discriminant",
         "cases"
     ],
-    "SwitchCase": [
+    SwitchCase: [
         "test",
         "consequent"
     ],
-    "TaggedTemplateExpression": [
+    TaggedTemplateExpression: [
         "tag",
         "quasi"
     ],
-    "TemplateElement": [],
-    "TemplateLiteral": [
+    TemplateElement: [],
+    TemplateLiteral: [
         "quasis",
         "expressions"
     ],
-    "ThisExpression": [],
-    "ThrowStatement": [
+    ThisExpression: [],
+    ThrowStatement: [
         "argument"
     ],
-    "TryStatement": [
+    TryStatement: [
         "block",
         "handler",
         "finalizer"
     ],
-    "UnaryExpression": [
+    UnaryExpression: [
         "argument"
     ],
-    "UpdateExpression": [
+    UpdateExpression: [
         "argument"
     ],
-    "VariableDeclaration": [
+    VariableDeclaration: [
         "declarations"
     ],
-    "VariableDeclarator": [
+    VariableDeclarator: [
         "id",
         "init"
     ],
-    "WhileStatement": [
+    WhileStatement: [
         "test",
         "body"
     ],
-    "WithStatement": [
+    WithStatement: [
         "object",
         "body"
     ],
-    "YieldExpression": [
+    YieldExpression: [
         "argument"
     ]
+};
+
+// Types.
+const NODE_TYPES = Object.freeze(Object.keys(KEYS));
```

Comment:
Did you mean to freeze an array here? I don't think this is necessary because we aren't exporting `NODE_TYPES`.

Reply:
I've stopped freezing it. I just copied it over from the index page where it also wasn't been used, so yeah, can be removed.